rust/xaynet-server/src/metrics/mod.rs

```diff
@@ -86,6 +86,30 @@ macro_rules! event {
 /// ```
 #[macro_export]
 macro_rules! metric {
+    (accepted: $round_id: expr, $phase: expr $(,)?) => {
```

Comment:
i'll move them. a side effect is that they can't have the same `metric!` name anymore, because both macros live in the same global `crate::` namespace.

Reply:
>  a side effect is that they can't have the same metric! name anymore
i think that is fine 👍